src/jti_background.js

```diff
@@ -61,6 +64,30 @@ function saveTemplates (templateJSON, callback, responseData = null) {
     });
 }
 
+function getDomains (callback) {
+    var domainList = {};
+    // Get the default domains
+    var domainListDefault = $.map(DefaultDomainList, function (domain, index) {
+        domain.default = true;
+        return domain;
+    });
+    // Get the custom domains
+    chrome.storage.sync.get(StorageID, function (data) {
+        if (data[StorageID]) {
+            var domains = data[StorageID].options.domains;
+            domainList = $.map(domains, function (domain, index) {
+                domain.default = false;
+                return domain;
+            });
+            // Sort them
+            domainList = utils.sortArrayByProperty(domainList, 'name');
+            // combine so that the default entries are always at top
+            domainListDefault = domainListDefault.concat(domainList);
+        }
+        callback(true, null, domainListDefault);
```

Comment:
I think it would make sense to swap `domainListDefault` and `domainList`
At a glance returning `domainListDefault` seems odd. We should be return `domainList`

Reply:
:+1: